content/admin/policy_mgmt/VulnerabilityManagement.md

```diff
@@ -0,0 +1,67 @@
+
+<a href="assets/images/Add-remediation-policy.png" rel="lightbox" title="Add Remediation Policy"><img class="thumbnail" src="assets/images/Add-remediation-policy.png"/></a>
+
+### Triggers
+
+In the **Triggers** section, use the checkboxes to select the time-based and/or route-based trigger. If a time-based trigger is selected, also input the number of days after which the trigger should fire.
+
+> **Note:** When using a policy with a route-based trigger, it is recommended to also use define a time-based trigger to account for those vulnerabilities which have been remediated in such a way that cannot be associated back to the original finding in Contrast. Typically, these cases only arise when the code was deleted, and therefore cannot be re-exercised, or redefined such that it occurs on a different route.
+
+If multiple policies affect the same vulnerability, the following rules determine Contrast's course of action:
+
+* Between two time-based triggers, the action with the closest deadline applies first. For example, if a violation deadline applies first, the vulnerability is flagged and then auto-verified when the later deadline applies. 
+* Auto-verification policies take precedence over violation policies. For example, if an auto-remediation deadline applies first, the vulnerability is closed and never flagged. 
+
+> **Note:** If Contrast rediscovers a legitimate vulnerability that was auto-remediated, Contrast will [reopen the vulnerability](user-vulns.html#analyze) as usual. 
+
+### Edit Policies 
+
+Click the name of an existing policy in the grid to see its detail view and update it. Click the trashcan icon in the grid to permanently delete the policy. Once deleted, Contrast won't notify you of vulnerabilities that were within the scope of this policy. You can also use the toggles in the grid to enable or disable individual policies, if you’d like to re-use the policy again later. 
+
+## Policy Violations and Auto-Verifications 
+
+If a Violation policy has been triggered, Contrast adds a warning note to the Vulnerabilities thermometer in the dashboard and flags the associated vulnerabilities in the Vulnerabilities grid. Hover over the warning icon for each vulnerability in the grid, or go to the vulnerability details page for more information about the violation.
+
+If an Auto-Verification policy has been triggered, its status is updated to Remediated - Auto-verified in the Vulnerabilities grids. Hover over the status text, or go the auto-generated comment in the vulnerability's **Discussion** tab for more information about the applicable policy. 
```

Comment:
Last I recall, there was some issues around whether we'd be able to deliver the discussion comments for Q4. We'll have to see where that ends up. Just something to be aware of for now though.

Reply:
Hi @dpcsgh, thanks so much for your comments. So the Discussion piece in the UI may or may not make it? I read a little bit about it, but I'm still lacking some context (just from not being around long enough). Also, just to clarify, this goes out with Dec release? Also still learning release schedules. :) Thanks!

Reply:
@gdvdsn-contrast I followed up, and it seems it's been completed, but is just on an un-merged branch presently. So, it should be good to include this then.